fabric-content-registries-v0/src/main/java/net/fabricmc/fabric/mixin/content/registry/OxidizableMixin.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.content.registry;
+
+import com.google.common.collect.BiMap;
+import com.google.common.collect.HashBiMap;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+import net.minecraft.block.Oxidizable;
+
+@Mixin(Oxidizable.class)
+public interface OxidizableMixin {
+	@Inject(method = "method_34740", at = @At("RETURN"), cancellable = true)
```

Comment:
This is a lambda body, right? If so, a comment or a `Dynamic` would explain that random intermediary method for maintainers (this is also done in other modules)

Reply:
I meant more like explanation in the `Dynamic` like in https://github.com/FabricMC/fabric/blob/fa34ed2c10c399ed6c973a56f3fda31cd47811f6/fabric-entity-events-v1/src/main/java/net/fabricmc/fabric/mixin/entity/event/LivingEntityMixin.java#L142